icu4c/source/tools/genccode/genccode.c

```diff
@@ -127,7 +129,8 @@ main(int argc, char* argv[]) {
         fprintf(stderr,
             "\t-o or --object      write a .obj file instead of .c\n"
             "\t-m or --match-arch file.o  match the architecture (CPU, 32/64 bits) of the specified .o\n"
-            "\t                    ELF format defaults to i386. Windows defaults to the native platform.\n");
+            "\t                    ELF format defaults to i386. Windows defaults to the native platform.\n"
+            "\t--skip-dll-export   prevent symbol export when statically linking ICU data\n");
```

Comment:
more of a nit, but maybe consider something like this (since the code refers to the symbol as the "entrypoint"):
"Don't export the ICU data entry point symbol (for use when statically linking)"

Reply:
Sure, I'll update the PR tomorrow.

Reply:
Done.